expression/builtin.go

```diff
@@ -247,6 +247,10 @@ func (b *baseBuiltinFunc) getArgs() []Expression {
 	return b.args
 }
 
+func (b *baseBuiltinFunc) setArgs(args []Expression) {
+	b.args = args
```

Comment:
If you call this method. You'd better clear the `hashcode` field. And this is the exact reason why i don't want to add the `set` method...